tasks/pipeline.py

```diff
@@ -9,12 +9,52 @@
 
 @task
 def trigger(ctx, git_ref="master", release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"):
-    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch)
+    """
+    Trigger a deploy pipeline on the given git ref.
+    The --release-version-6 and --release-version-7 options indicate which release.json entries are used.
+    To not build Agent 6, set --release-version-6 "". To not build Agent 7, set --release-version-7 "".
+    The --repo-branch option indicates which branch of the staging repository the packages will be deployed to.
+
+    Example:
+    inv pipeline.trigger --git-ref 7.22.0 --release-version-6 "6.22.0" --release-version-7 "7.22.0" --repo-branch "stable"
+    """
+    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch, deploy=True)
+    wait_for_pipeline("DataDog/datadog-agent", pipeline_id)
+
+
+@task
+def run_all_tests(ctx, git_ref="master", here=False, release_version_6="nightly", release_version_7="nightly-a7"):
+    """
+    Trigger a pipeline on the given git ref, or on the current branch if --here is given. This pipeline will run
+    all tests, including kitchen tests.
```

Comment:
nit: to make the `inv --list` output better.
```suggestion
    Trigger a pipeline on the given git ref, or on the current branch if --here is given.
    This pipeline will run all tests, including kitchen tests.
```